app/models/user.rb

```diff
@@ -20,7 +20,7 @@ class User < ApplicationRecord
 
   validates :provider, presence: true
   validates :uid, presence: true, uniqueness: { scope: :provider }
-  validates :nickname, uniqueness: true, if: :nickname?
+  validates :nickname, presence: true, uniqueness: true
```

Comment:
Could we add the validation on the DB layer too?
I have updated all users nicknames, now we don't have empty values:
```ruby
irb(main):054:0> User.where(nickname: [nil, ""]).count
D, [2021-08-05T08:38:34.316870 #1628] DEBUG -- :    (0.7ms)  SELECT COUNT(*) FROM "users" WHERE ("users"."nickname" = $1 OR "users"."nickname" IS NULL)  [["nickname", ""]]
=> 0
```